src/MixpanelEventForwarder.js

```diff
@@ -245,8 +284,20 @@ var constructor = function () {
     this.process = processEvent;
     this.setUserAttribute = setUserAttribute;
     this.setUserIdentity = setUserIdentity;
-    this.onUserIdentified = onUserIdentified;
     this.removeUserAttribute = removeUserAttribute;
+
+    this.onLoginComplete = onLoginComplete;
+
+    // For all Identity Requests, we run mixpanel.identify(<device_id)
+    // as per Mixpanel's documentation https://docs.mixpanel.com/docs/tracking-methods/identifying-users
+    // except when a user logs out, where we run mixpanel.reset() to
+    // detach the distinct_id from the device_id
+    this.onLogoutComplete = onLogoutComplete;
+
+    // For these two methods, we are essentially passing along the
+    // Identify request the same way.
+    this.onIdentifyComplte = sendIdentifyRequest;
+    this.onModifyComplete = sendIdentifyRequest;
```

Comment:
Is it confirmed that `onModifyComplete` should send a `mixpanel.identify` request?  I thought if we run identify twice with 2 different identities, that we would get that error.